sensors/06-list-datasets-by-plot.Rmd

```diff
@@ -85,7 +85,7 @@ dataset = get_file_listing(None, url, key, sensor, sitename,
 ```
 
 
-# Querying the API
+## Querying the API
```

Comment:
maybe comment out this entire section?

Reply:
I can, I didn't actually look at this chapter in terms of contentreally, just fixed the header levels and set `eval = FALSE` on all the chunks.

Reply:
Its low priority ... @KristinaRiemer what do you think?

Reply:
I think it's worth having the curl method available in the tutorials, as long as the commands work.